src/main/msp/msp.c

```diff
@@ -1833,7 +1829,7 @@ case MSP_NAME:
 
         // API 1.41 - Add multi-gyro indicator, selected gyro, and support for separate gyro 1 & 2 alignment
         sbufWriteU8(dst, getGyroDetectionFlags());
-#ifdef USE_MULTI_GYRO
+#if GYRO_COUNT > 1
         sbufWriteU8(dst, gyroConfig()->gyro_to_use);
```

Comment:
Configurator can use this value to detect that more than 2 gyros are supported. But this packet is already mess.

It is probably better to report gyro 0,1 here for backward compatibility and create new packet for gyro reporting (with support for at least 8 gyros, using masks)